### ISSUE TITLE
Strange output log

### ISSUE THREAD

**Issue by launchauto** [zdaxie/Transformer-Self-Supervised#2]
Hi authors, I have pretrianed your moby_swin_tiny model using 8 Tesla V100 GPU
and reproduced your results in downstream task. I get 74.394% on linear evaluation and 43.1% on COCO object detection task, 39.3% on COCO segmentation task. But the loss and grad_norm is really weired during training. Can you show me your log?
Here is my log. The loss drops to 7 and then rises to 16, then never drop again. During the pretraining task, the grad norm average value sometimes rises to infinite.
[log_rank0.txt](https://github.com/SwinTransformer/Transformer-SSL/files/6557998/log_rank0.txt)

**Comment by launchauto** [comment]
The uploaded txt is one of the GPU pretrain log.

**Comment by launchauto** [comment]
The uploaded txt log_rank0.txt is one of the eight gpus pretrain logs.
And the uploaded txt log_rank7.txt is one of the eight gpus linear evaluation logs.
[log_rank7.txt](https://github.com/SwinTransformer/Transformer-SSL/files/6558096/log_rank7.txt)

**Comment by launchauto** [comment]
Here is my linear result. As you can see, though the pretrain log is somehow miserable, the final result make senses.

**Comment by michuanhaohao** [comment]
I also encountered the same problem.

**Comment by tbup** [comment]
@launchauto @michuanhaohao me too, but I run it with precision O0. Did you run with the O0 precision? 
[log_rank0.txt](https://github.com/SwinTransformer/Transformer-SSL/files/8021405/log_rank0.txt)

**Comment by Rocky1salady-killer** [comment]
我也遇到了这个问题！loss一直是16永远不会下降？

**Comment by Rocky1salady-killer** [comment]
怎么才能不适用apex混合精度呢？我使用swin transformer进行训练的时候，loss就会下降并且收敛。然而，我注意到swin transformer工程当中没有使用apex混合精度

**Comment by Chengyang852** [comment]
Is it normal for the loss value to be around 16?  Has anyone encountered this problem?

**Comment by YohjiNtpu** [comment]
> 怎么才能不适合用apex混合精度呢？我用swin transformer进行训练的时候，loss就会下降并收敛。不过，我注意到swin transformer工程中没有使用apex混

请问您的问题解决了吗

**Comment by YohjiNtpu** [comment]
> loss值在16左右正常吗？有没有人遇到过这个问题？



> loss值在16左右正常吗？有没有人遇到过这个问题？

我也是

**Comment by Pang-b0** [comment]
Excuse me, have you solved the problem that loss drops to 8.9 and then rises in the opposite direction? Is it caused by apex mixed precision training?

**Comment by YohjiNtpu** [comment]
> 请问，loss下降到8.9然后反方向上升的问题解决了吗？是顶点混合精度训练导致的吗？

没有/(ㄒoㄒ)/~~

**Comment by Pang-b0** [comment]
哈 会不会是loss函数的问题呀，这个代码你还在关注吗

**Comment by Pang-b0** [comment]
> 

会不会是loss函数的问题呀 这个代码你还在关注吗，我的loss从开始就是16 降不下去

**Comment by YohjiNtpu** [comment]
> > 
> 
> 不会是loss随便数的问题呀这个代号你还在关注吗，我的loss从开始就是16降不下

我也没有解决。。
